internal/errno/errno.go

```diff
@@ -192,36 +192,44 @@ var (
 	// 100: database/SQL (init failed)
 	ERR_INIT_SQL_DATABASE_FAILED = EC(100000, "init SQLite database failed")
 
+	// 108: database/SQL (execute SQL statement: disk table)
+	ERR_UPDATE_DISK_FAILED      = EC(108000, "execute SQL failed while updating disk")
```

Comment:
Those error codes should has `11` prefix, see [sub-category](https://github.com/opencurve/curveadm/wiki/errno#%E5%AD%90%E7%B1%BB)

Reply:
Done.